src/renderer/components/dock/terminal/create-terminal-tab.injectable.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable, lifecycleEnum } from "@ogre-tools/injectable";
+import dockStoreInjectable from "../dock/store.injectable";
+import { DockStore, DockTabCreateSpecific, TabKind } from "../dock/store";
+import { bind } from "../../../utils";
+
+interface Dependencies {
+  dockStore: DockStore;
+}
+
+export function createTerminalTab({ dockStore }: Dependencies, tabParams: DockTabCreateSpecific = {}) {
+  return dockStore.createTab({
+    title: `Terminal`,
+    ...tabParams,
+    kind: TabKind.TERMINAL,
+  });
+}
+
+const createTerminalTabInjectable = getInjectable({
```

Comment:
This bind pattern is just complexity which shouldn't be added. Configuration (e.g. dependencies) shouldn't be mixed with arguments. Arrow function would be much simpler.

Reply:
Inline `createTerminalTab`